### DISCUSSION TITLE
Update russian library

### DISCUSSION THREAD

**Discussion by fitwist** [explosion/spaCy#13128] [Language Support]
We've been using ru_core_news_sm to lemmatize set of posts and solve topic modeling problem. During the EDA, I've found non-lemmatized words and I'm ready to format this new words in order to update the word listings. Is it possible to make an update to the library?

Some of examples are (the data is IT related):
созвон
коммивояжёр
рендеринг
тестировщик
веб-приложений
веб-разработки
релокейт
бета-тест
криптовалюта
нейросеть
сниппет
репозиторий

**Comment by adrianeboyd** [comment]
The lemmas in `ru_core_news_sm` come from a third-party library `pymorphy3`, which provides dictionaries under the packages `pymorphy3-dicts-ru` and `pymorphy3-dicts-uk`.

It looks like the Russian dictionary comes from this repo, and it also looks like it should be possible to create a custom dictionary, but you'd have to look into how to get the spacy lemmatizer to load your custom dictionary instead of the default one:

https://github.com/no-plagiarism/pymorphy3-dicts

The default spacy lemmatizer for Russian is under `spacy/lang/ru/lemmatizer.py`.
